### DISCUSSION TITLE
Add WiFi Option?

### DISCUSSION THREAD

**Discussion by StodgyWaif** [mp-se/gravitymon#97] [Ideas]
Hello.
Is there any way you could add an option to show the stored WiFi settings?
Thanks.

**Comment by mp-se** [comment]
I've filtered out those due to security concerns since there is no real authentication for the API's so anyone on the network would be able to grab those secrets. You can get the wifi names configured if you get the raw config data from the API like http://yourdevice/api/config. 

Do you want to view both wifi and passwords ? Whats the reason for showing them ? Or do you want to edit them as well ?

**Reply by StodgyWaif** [reply]
Thanks for replying and for the information.
I have a secondary AP set up and I didn't know there was a way to see if I had previously configured both.
So I end up re-entering the WiFi settings every time I brew a new batch.
The api/config file worked perfectly though.
Thanks again.

**Reply by mp-se** [reply]
Perfect, i can add the info on configuered ap to the next release
